apps/juicebox_web/assets/js/reactions/reactions.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import { sendReaction } from './actions';
+import ReactionRecorder from './recorder';
+import ReactionPlayer from './player';
+
+class Reactions extends Component {
+  reactions() {
+    return map(this.props.reactions, (video, userId) => {
+      return <ReactionPlayer reaction={video} key={userId} />
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <ReactionRecorder onRecord={_send.bind(this)} />
+        <div className="reactions">
+          { this.reactions() }
+        </div>
+      </div>
+    );
+  }
+}
+
+Reactions.propTypes = {
+  reactions: PropTypes.array,
```

Comment:
Possible undefined variable: 'PropTypes' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)